src/platform_impl/linux/mod.rs

```diff
@@ -388,6 +388,26 @@ impl Window {
         x11_or_wayland!(match self; Window(w) => w.set_max_inner_size(dimensions))
     }
 
+    #[inline]
+    pub fn resize_increments(&self) -> Option<PhysicalSize<u32>> {
+        match self {
+            #[cfg(feature = "x11")]
+            Window::X(ref w) => w.resize_increments(),
+            #[cfg(feature = "wayland")]
+            Window::Wayland(_) => None,
```

Comment:
Could you add direct calls to Wayland? I'll add support for it later on, so we can at least has a warning for now.